client/src/components/googleMap/googleMap.jsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Loader} from '@googlemaps/js-api-loader';
+import './googleMap.sass';
+
+/* eslint-disable import/no-unresolved */
+const config = require('../../../../server/config');
+/* eslint-enable import/no-unresolved */
+
+const loader = new Loader({
+	apiKey: config.mapsAPIKey,
+	version: "weekly"
+});
+
+export class GoogleMap extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			markers: [],
+			location: null
+		};
+
+		this.mapRef = React.createRef();
+		this.createDisplayMap = this.createDisplayMap.bind(this);
+		this.createSightingMap = this.createSightingMap.bind(this);
+		this.createMarkers = this.createMarkers.bind(this);
+		this.addMarker = this.addMarker.bind(this);
+		this.showMarkers = this.showMarkers.bind(this);
+		this.hideMarkers = this.hideMarkers.bind(this);
+		this.deleteMarkers = this.deleteMarkers.bind(this);
+		this.updateLocationMarker = this.updateLocationMarker.bind(this);
+		this.addEventListener = this.addEventListener.bind(this);
+	};
+
+	//Creates a new marker on the map, updates state with the new marker
+	addMarker(location, map) {
+		const newMarker = new google.maps.Marker({
+			position: location,
+		});
+
+		let marker = this.state.markers;
```

Comment:
this can be `const`

Reply:
I'm guessing you mean the `let marker` turns into `const marker`

Reply:
Sorry - sometimes the line specific stuff isn't always the most obvious.